.jenkins/Jenkinsfile

```diff
@@ -506,13 +376,10 @@ def saveAgentReport() {
 /////////////////////////////////////////
 
 def generateUnifiedTestReport() {
-  assert "Summary".equals(STAGE_NAME)
-  node('cassandra') {
-    checkout scm
-    sh "mkdir -p build"
+  node(getNodeArch('amd64')) {
     copyArtifacts filter: 'test/output/**/*.xml.xz', fingerprintArtifacts: true, projectName: env.JOB_NAME, selector: specific(env.BUILD_NUMBER), target: "build/"
     sh 'xz -d build/test/output/**/*.xml.xz || true ; xz -d test/output/cqlshlib.xml test/output/nosetests.xml || true'
-    sh "ANT_OPTS='-Xmx15g' ant generate-test-report"
+    sh ".build/docker/_docker_run.sh bullseye-build.docker generate-test-report.sh"
     dir('build/') {
       sh "xz -f TESTS-TestSuites.xml"
       archiveArtifacts artifacts: "TESTS-TestSuites.xml.xz,test/html/**/*", fingerprint: true
```

Comment:
we'll need to make a zip file that matches apple's ci results bundle here…

Reply:
Can we have a quick call to discuss it?